src/app/search-playground/search-playground.component.scss

```diff
@@ -0,0 +1,8 @@
+.expert-search-container {
+  min-height: 100%;
+  min-width: 100%;
+}
+
+.mat-input-element {
+  font-family: 'Courier New', Courier, monospace
+}
```

Comment:
Same as in [ projects/dsp-ui/src/lib/search/expert-search/expert-search.component.scss](https://github.com/dasch-swiss/dsp-ui-lib/pull/114/files#diff-85ff81ac3b3179331a4d85ae475d026f) line 21-23
It doesn't need the same style here again?

Reply:
done in 0e81d2c